sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationClient.java

```diff
@@ -710,24 +864,26 @@ public List<TranslatedTextItem> translate(
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public List<TranslatedTextItem> translate(List<String> to, List<InputTextItem> content) {
+    public List<TranslatedTextItem> translate(List<String> to, List<InputTextItem> requestBody) {
```

Comment:
Seems almost all changes in client class is name of `content` -> `requestBody`...

Reply:
It is caused by the change we did in TypeSpec. I can find details why it was done but I believe it was one of the feedback we got during the intial review - "content" is not good enough word.